test/new_relic/agent/method_tracer_helpers_test.rb

```diff
@@ -13,6 +13,19 @@ def private_method; end
   end
 end
 
+class ApplicationController; end
+class TestController < ApplicationController; end
+
+class String
+  def underscore
+    'test_controller'
+  end
+
+  def join(*args)
+    'User/fake_app/app/controllers/test_controller.rb'
+  end
+end
```

Comment:
Cool idea! Are these methods still being used? If so, can you show me how they work?